### DISCUSSION TITLE
关于 「不要再被头顶冒火欺骗了，用始皇的 API 服务吧」的声明以及本站未来发展方向

### DISCUSSION THREAD

**Discussion by BennyThink** [BurnHair/docs#144] [网站公告]
今天早上朋友给我发了一个链接 [https://linux.do/t/topic/126124](https://linux.do/t/topic/126124) [存档](https://web.archive.org/web/20240721054030/https://linux.do/t/topic/126124)，说我被人挂了

看了一下，是一个名叫 `ishane1112` 的人说我无故封号。

> 前天突然发现不能用了，才发现号被封了，我都不知道自己干了啥就被封号，联系他们家Telegram，写邮件一律没有回应，我还有二十多美金在里面，连个交代都没有，这跟抢钱有什么区别。
> 
> 
> 我也不差这点钱，穷不了我，也希望坛友以此为警示，有钱的赶紧用，没钱的也别去充钱了，搞不好就是跟我一样的下场。
> 

这个用户我眼熟，邮件我看到了但是没回，Telegram 的消息也没回，这些都是我故意的，因为我觉得没有必要和无理取闹的人讲道理。就算我贴出来了证据，他也会说这些都是我伪造的
<img width="617" alt="image" src="https://github.com/user-attachments/assets/70f70f6d-4cdf-4531-b7c1-85f68ae49198">

# 封禁原因

我先说说封他号的原因：**重复注册**、**刷邀请**、**恶意诽谤**

从 6 月初开始，我开发了一个丢骰子的功能。后来为了增加透明度，还把[随机数的计算方式公开](https://t.me/mikuri520/2832)，方便使用者去验证，来表明我没有从中作梗。

> 说到丢骰子这里，还有个人趁着非 VIP 分组不能丢骰子去做了验证 [https://linux.do/t/topic/125354](https://linux.do/t/topic/125354) 
>
> 我为什么要做出只有 VIP 用户才能丢骰子的决定呢？因为白嫖的人太多了，一个人注册一堆小号，而每次丢骰子都会有 $0.5 的平均收益。

某天，突然发现他发了这样的帖子 [https://linux.do/t/topic/122334/6](https://linux.do/t/topic/122334/6)

<img width="826" alt="Untitled 1" src="https://github.com/BurnHair/docs/assets/14024832/5b3aead0-8070-4382-b145-d6196c77ef2a">

> 就这个数据，我不信这个丢骰子是随机的，钱用完不用他家了，用始皇的，我相信始皇的人品。。。
> 

# 真的吗？

我看了下他的丢骰子记录，结果发现他有的赚

<img width="1046" alt="Untitled 2" src="https://github.com/BurnHair/docs/assets/14024832/5202f58e-e76f-4b6d-be90-c5f78c318578">

这个页面不是很直观，那么直接看我从数据库里拿出来的结果吧

```python
{1: 3, 2: 3, 3: 3, 4: 1, 5: 5, 6: 1}
```

用 Excel 画个图

<img width="776" alt="Untitled 3" src="https://github.com/BurnHair/docs/assets/14024832/84e36a58-c786-4498-adec-18a9898e53a5">

一共投掷 16次，收益 $5，虽然离理论收益有**一点点差距**（因为样本数量太少了），但是这至少是不亏的吧？

于是又去翻了下他的充值记录，支付方式支付宝 ：充了 $3 

<img width="1544" alt="Untitled 4" src="https://github.com/BurnHair/docs/assets/14024832/82fcb468-5bbb-441b-9ad3-5c96db6fa24d">

在我这里记录的总消费 $44
<img width="703" alt="Untitled 5" src="https://github.com/BurnHair/docs/assets/14024832/12c58ee5-6dde-41e9-af08-46c8d991ec9f">

划重点，
> 我还有二十多美金在里面，连个交代都没有，**这跟抢钱有什么区别**。

说得好像这 20 多是自己充值的一样，原来只充了3块啊

这还没完，我就又翻了下他的邀请记录，发现他邀请了很多账号，id 是接近甚至连续的

<img width="604" alt="Untitled 6" src="https://github.com/BurnHair/docs/assets/14024832/94c39dfb-c18f-4e05-ab29-4824413bac25">

看下被邀请的用户的呢？竟然还有重复的IP的，更别提还有其他更相似的特征

<img width="784" alt="Untitled 7" src="https://github.com/BurnHair/docs/assets/14024832/36351455-f5ea-45a1-bc9c-989b401ebf1e">

这些 GitHub 账号的注册日期基本都在 6月3日前后
<img width="934" alt="Untitled 8" src="https://github.com/BurnHair/docs/assets/14024832/7cbfc0a6-a830-4da0-af3f-4be45740f679">

正常人都会觉得，这事一定不正常吧？哪有那么巧，一堆连着的 ID 都被这个人邀请，然后还恰巧都是新注册的 GitHub 账号？

更有趣的是，有很多账号签到时间还很接近，比如这个吧
![Untitled 9](https://github.com/BurnHair/docs/assets/14024832/c550a884-4e0f-4012-a668-07e3150ced1c)

然后这些号 0 使用记录，让我不自禁的想象这是在养号要以后备用的吧。

所以我的疑问就是：

1. 以 1 折不到的价格享受了福利，还阴阳怪气
2. 只贴亏的部分不贴赚的，怀疑人品
3. 不好用还契而不舍的请求，32.2K的请求数量可不低啊
4. 不好用还邀请自己
5. 不好用还发两封邮件、留两条消息。为了一个不好用的东西这么费力真的是太难为人了

回到我封禁的理由：**重复注册、刷邀请、恶意诽谤**。我觉得我这样做没有错。

说完了这些我从数据库里拿出来的客观事实，再看一下我从他的历史发言里看到的信息

- 到底能不能分享下怎么 C 号的，痒痒
- 我 C 成功了，来报喜，见图

<img width="475" alt="Untitled 10" src="https://github.com/BurnHair/docs/assets/14024832/1882a1b8-813e-4233-928d-1b001512c318">

在这个论坛里，C 表示的是 crack 破解的意思，我还是从这个 [Twitter](https://x.com/DerekNee/status/1783064415048491428) 里学习到的。

当然了，怎么评价他个人的发言，每个人心里都有不同的标准。作为一个理性的人，我会尽可能的避免用我的主观想法去影响我的判断。在封号与否这件事情上，我会尽量要求自己只用客观数据作为决策依据。

写了这么多，我也不知道该说些什么。 

# 我的反思

我对于重复注册、刷邀请是零容忍的，在注册登录的用户协议里也明确表明：

1. 每人只允许注册并拥有一个账号
2. 如果发现任何违反注册规则的行为，我们有权在不另行通知的情况下封禁相关账号

我也知道存在误封，在用户询问的时候，如果态度诚恳、解释充分、并且我数据不是十分充足的情况下，我会给解封的。每个人都会犯错，也应该得到一次改进的机会。

今天我就一直在想，我做这一切是图了什么？我有自己的生活，有工作，每天有很多有意义的事情要去做，为什么要把宝贵的时间投入到与这些傻逼的较量之中？

从开始做这个站的时候，遇到过这样的

![Untitled 11](https://github.com/BurnHair/docs/assets/14024832/bc30cbf2-40ba-4263-9a1d-8686bf3d8b10)

还有这样的

![Untitled 12](https://github.com/BurnHair/docs/assets/14024832/3ca87e27-22fe-4f90-aaea-f68f2c8dd6be)

以及这样的
<img width="614" alt="Untitled 13" src="https://github.com/BurnHair/docs/assets/14024832/a133094c-4243-413c-8abf-acb4cdbb5f83">

更别提还有人大量 NSFW 导致我的账号被封等

<img width="1243" alt="Untitled 14" src="https://github.com/BurnHair/docs/assets/14024832/6a0293ab-7bde-4d2e-87f2-2b2a53224ee2">

做公益有用吗？我这里的教训和答案是没用。**升米恩，斗米仇**，给的东西不好还会被骂，被滥用，被拿去做黑产，被拿去给别人做嫁衣，连一句谢谢都未必能听到。

# 未来的产品策略

互联网这个东西很神奇，它大大降低了知识获取的难度，互联网改变了我们获取和分享知识的方式，使得信息更容易、更快速地流通。无论是通过搜索引擎找到所需信息，还是在各种教育平台上学习新技能，互联网都极大地扩展了我们的学习边界和可能性。

在AI的时代也同样如此。对于 OpenAI 及 ChatGPT，我深知在国内想要正常使用有多艰难。不仅要翻墙，还要找到好的IP，而且还需要解决海外银行卡的问题。我也有很多朋友，他们也很想试试用 AI 提高生产力，可是他们并做不到。

这也是一开始我们的定位是公益 + 付费的原因，缩小数字鸿沟，让所有人都能够利用 AI 有效的学习、发展、进步。

可事实呢？还是我们想得太简单、太轻松了，这种乌托邦的想法很不切实际。

1. 免费用户滥用太多，影响到付费用户的体验了，谁不吃饭，难道自己不吃饭做慈善吗？

2. 小号太多，被其他人牟利，真正的免费用户特别少，管理精力耗费严重

3. 资源有限

在整件事情上，我们的策略、对用户的定位、产品价值和自身的定位都出现了不少的判断失误，也因此给自己惹了很多麻烦。

--------------------

因此，在经过讨论之后，我们不得不做出一个艰难的决定：

我们将改变 「头顶冒火」运营方向：**以付费用户为核心**，优先保障付费用户的使用，为付费用户提供一定量的优惠与折扣；与此同时，所有公益免费用户的服务，只能尽力而为。

恰如在新的用户协议中提到的一样，我们将按照计划逐步推进新的产品策略：

- 公益额度（适用于初次注册、签到、参与活动）：为了防止对免费服务的滥用，本类额度不提供任何稳定性或服务承诺。在资源有限或管理精力不足时，本站保留随时回收公益额度的权利。
- 奖励额度（适用于邀请他人、进行宣传活动）：作为对用户对本站的贡献，此类奖励永久有效，但不适用于任何消费折扣。
- 充值额度：余额永久有效，本站将优先保证充值余额的资源和稳定性供应。在任何情况下，包括需要退款时，都将确保用户权益得到保障。

<aside>
💡 付费用户不用担心，付费额度都会顺利转移过去，一分都不会少。
</aside>

如要继续使用，请已有用户**尽快同意**最新的服务条款；在通知期结束之后如未同意可能会影响正常使用。

**Comment by ZEERDEER** [comment]
确实傻逼，我就签到，后期丢骰子现在都90多刀了… 他那不就是赌徒吗？输了就撒泼，傻逼一个

**Reply by p33333333333333l** [reply]
虽然🎲透明，但是我自己丢的时候确实非酋附体，还是老老实实签到了😭
对本次事件作为一个免费用户我只能说：老号别搞🥺

**Reply by GuoJiaxing1997** [reply]
站长加油，我们家的使用体验和售后还是很好的，尤其是节日的各种小活动！
——来自一个连续三次丢骰子丢出1点的非酋用户

**Comment by Equuuu** [comment]
我是网站的受益者  感谢付出

**Reply by boyivip999** [reply]
我也是受益者

**Comment by everstu** [comment]
我应该属于第一批用户了，看着Benny的头顶冒火一路的经历。

从开始的注册送5刀，到现在的注册送1刀，我想说，有些人是真特么恶心啊。

我是网站的受益者 感谢付出，永远支持Benny。

如果是真是因为钱多恶意封号，那我肯定是首当其冲，我邀请几十人，余额多到离谱。

**Comment by Feiyuyu0503** [comment]
用了很久Benny的服务了，从很久以前就关注Benny各种telegram bot教程，感谢付出。

**Comment by xiaotianxt** [comment]
支持一波 Benny，纯福利服务怎么还有人不满意。自从用了 burn.hair，我的 openai API 账户都没付过款。日常使用额度完全够用，简直太良心了好吗🫶

**Comment by Lion066** [comment]
OpenAI API账户被封以后不想再折腾了，甚至放弃使用，直到用了 burn.hair，又可以继续使用API了

**Comment by zakizou** [comment]
白嫖用户一直体验都很好，后续要是不好用了，还是很遗憾啊

**Comment by GinWU05** [comment]
目前是白嫖，因为够用，但后续会是付费用户，感谢Benny付出！

**Reply by GinWU05** [reply]
不充不能摇骰子了，我先冲了！！

![image](https://github.com/BurnHair/docs/assets/26453811/ed998077-5267-4f74-bf27-ca5c4cba8c26)

**Comment by shuangmuaigwu** [comment]
感谢付出

**Reply by shuangmuaigwu** [reply]
经典我白嫖你的你就得让我好好白嫖，不让我嫖我可就开始闹了啊哈哈哈，真的小学生行为脑残

**Comment by xxclbkmmx** [comment]
白嫖用户路过 非常感谢无私付出

**Comment by strive-97** [comment]
看得我血压都升高了，我是第一批使用者，感谢付出

**Comment by ShuSongXiao** [comment]
目前签到积累到了50多刀，因为平常用得少，只有遇到问题时才会问问。后面如果需求多了也会付费，支持大佬

**Comment by zhangdeshuai409930360** [comment]
白嫖用户路过，哈哈～

**Reply by ShuSongXiao** [reply]
帅哥，你有多帅

**Comment by Bernankez** [comment]
看了眼论坛，这种人确实没有什么回复的必要，你回复了它还是会嘴硬的

**Comment by IceCodeNew** [comment]
支持，不要把精力浪费在不值得的人身上~

**Comment by EFLKumo** [comment]
笑死我了
![image](https://github.com/BurnHair/docs/assets/96906415/6ed3d708-eabf-4b2a-9802-a1a4253582e1)

**Comment by PlayMcBKuwu** [comment]
如果這個人只是 C(rack)，去搞搞外國的服務，那很正常。
整個論壇目前都是這個走向，在 LINUX DO 隨便 C 幾個服務無可厚非。
不如說論壇和「秦始皇」以及 OAIPro 并沒有做錯什麼，因為在 LINUX DO 是強烈抵制 C 國內的公益或者付費服務……只能說實在是急了，於是就發了個混淆視聽的話題，明顯能看出在濫用後還在惡意誹謗。
由於自己也是 Burn Hair 的收益者，在此感謝 Benny 無私提供公益服務。

**Reply by EFLKumo** [reply]
捉个氨糖

**Comment by lydong0925** [comment]
支持站长

**Comment by nmdwgll** [comment]
烂人太多了, 果断充值支持下Benny

**Comment by fokx** [comment]
支持Benny小土豆

**Comment by xiaopalu-max** [comment]
我靠，这人确实恶心，要是都这样搞的话 以后估计都不会有这样的良心网站了，我支持Benny小土豆！

**Comment by check321** [comment]
支持Benny

**Comment by haowei93** [comment]
哈哈， 白嫖的流量够我用很久了， 后面 一定充值成为尊贵的 vip

**Comment by farion1231** [comment]
感谢Benny的无私付出，已成为付费用户。

**Comment by Flyinsky2004** [comment]
把他人的善心当作理所当然，只能说这样的人不配享受公益服务，我从burnhair在ehco开放第一天就注册了，当时真的是十分感激为大家做公益的人，结果不到一天就被恶意薅炸了，真的人心险恶。现在claude,openai官网也都放出了限时限量的模型使用，这些api的使用方式也很少用了，不过还是非常感激大佬为我们所作的一切。祝好。

**Comment by imldy** [comment]
大号白嫖三十多刀了，虽然之前也注册了小号，但被封之后就坦然接受了（这也应该是正常反应，毕竟不封是手下留情，封了是正常操作），没想到竟然有人发帖谴责。

**Comment by Fryflykid** [comment]
白嫖了30多刀，非常好用，也被误封过一次，后续立马解封了。用完这30刀后面一定会付费。非常感谢Benny，给我的工作带来很大的便捷

**Comment by jkjoy** [comment]
很好用

**Comment by tarkyn915** [comment]
一直白嫖的普通用戶，確實幫助了我很多。 下次有折扣會支持！

**Comment by dabiao2008** [comment]
我的使用量不大，也就是简单的折腾，测试一下gpt新模型是不是更先进了
但是薅羊毛谁不爱呢。
天天签到获得额度，已经达到30刀了。
准备充值3刀，支持Benny继续运营下去。

**Comment by daixd2007** [comment]
感谢支持Benny

**Comment by ZZZZihan** [comment]
支持benny

**Comment by acshameless** [comment]
一直白嫖中，充值一波，thx

**Comment by fewk** [comment]
用得少，签到累积30多刀了。

我觉得源头是骰子，之前连输让我心态不好，我还是挺反感赌的行动，会让人心态不好。我发现后我也不再玩骰子了。站长可以考虑一下撤下这功能。这骰子就是会引来一些烂人，也会让人成为烂人。赌毒不是好东西。。。

很多波菜站就是这样，开头给点高概率，吸引进来，一些人输了后人心态就变了，没有定力了。行为会越来越疯狂。输赢对他们不是重点，输入了想回本，赢了想赢更多，开奖过程带来的刺激才是爽点。

如同这人，骰子连输几次心态崩了，发贴骂，即便人只投入签到的少少时间成本。赢了也会想着开小号赢更多。获得更多的余额是他的爽点，用不用是其次。

**Reply by BennyThink** [reply]
下线了！

**Reply by farion1231** [reply]
感谢Benny，骰子我还玩的挺开心的...

**Reply by BennyThink** [reply]
> 感谢Benny，骰子我还玩的挺开心的...

说明你赚了！

**Reply by HengMrZ** [reply]
哎。我有输有赢玩的也挺开心的

**Comment by departever** [comment]
白嫖用户路过，感谢Benny站长的付出！只能说辛苦了

**Comment by HBUT-yang** [comment]
白嫖的路过，感谢付出！！！

**Comment by anghunk** [comment]
免费用户路过，感谢付出，准备转付费用户~

**Comment by adriftliu** [comment]
这很中国哎
感谢Benny

**Comment by cz265** [comment]
谢谢 支持Benny的付出

**Comment by aBner6667** [comment]
感谢付出，无论你怎么选择，都能理解。

**Comment by craigary** [comment]
看完这条消息终于理解为啥网站叫「头顶冒火」了 🫠

对于个人用户，靠签到获得的 Credit，完全足够日常使用了。真的感谢 Benny 老师的付出，希望薅羊毛的少一点，能持续运营下去 🙏

**Comment by bxb100** [comment]
充了 3 刀, 到现在也就用了 1 刀 =. =

**Comment by ChandlerCS** [comment]
沉默的大多数过来感谢下站长，爱你呦！

**Comment by lydong0925** [comment]
支持 佬

**Comment by HengMrZ** [comment]
就说为啥投不了骰子了。。
你的剩余额度为：公益17225963，奖励 1000000，充值12000000，总计折合$60.45

**Comment by luxingchang2024** [comment]
你好，我的用户为啥被封禁了，

**Reply by luxingchang2024** [reply]
我啥也没有干。偶尔用户。更没有滥用。结果给我封号了。请楼主帮忙解封。

**Comment by XCJYO** [comment]
从最开始注册就送100刀，到50刀，10刀，到现在越来越严苛，都是因为有许多人恶意白嫖，恶意白嫖的资源远远大于那些正儿八经使用的，在这里也感谢大佬能让我用上最前沿的chatgpt-4o，以后工作了也会多支持大佬

**Comment by tianlichunhong** [comment]
为什么把我号封了？我啥也没有干。偶尔用户。更没有滥用。结果给我封号了。请楼主帮忙解封。不信，看我的使用记录。虽然使用不多，一直免费使用，还是谢谢楼主。

**Reply by tianlichunhong** [reply]
> 为什么把我号封了？我啥也没有干。偶尔用户。更没有滥用。结果给我封号了。请楼主帮忙解封。不信，看我的使用记录。虽然使用不多，一直免费使用，还是谢谢楼主。

账号是恢复了。之前签到的金额给清零了。现在签到需要绑定telegram，还是通过手机号码。要获取隐私信息就算了。国内的国宝太需要这样的信息了。之前通过国内电话号码注册的电报账号。因为是电报是通过短信验证的，国宝们都一一登记。随时找你喝茶。虽然不会干啥事情，为了这点用量，不值得。

**Reply by BennyThink** [reply]
真是搞笑，还来问为什么被封号，因为你注册了三个小号，自己不知道吗？
更搞笑的是，你输入手机号时，手机号是发给 Telegram的并不是我，不信你自己开F12看。你都说了不值得，那确实应该满足你的愿望👍

**Reply by ip520011** [reply]
清额度这个受不了，好不容易攒点额度，一下给清完了

**Comment by luxingchang2024** [comment]
@BennyThink 
你好，为啥我的账户被封禁了，我啥违规操作也没有做，也就是每一天坚持打卡，为什么突然一天封号了，也不做什么解释。

**Comment by mrjesen** [comment]
赞成因为重复注册封号，但是封号直接永久封，并且不通知还是挺烦的

**Comment by cn1860** [comment]
我是来感谢的。一开始是白嫖的，我承认我也邀请过自己，不过都被封了，没有怨言，尊重规则，也尊重冒火。ai带给我很大的方便，但是诸多门槛限制了我，幸好有冒火，我一直稳定使用，再次感谢，同时也对我之前偷鸡摸狗的自我邀请道歉。我现在是付费用户，这个世界应该有规则，我也应该为别人的付出支付报酬。我知道还有别家，但是我信得过这里。

**Reply by farion1231** [reply]
给你点个大大的赞！

**Comment by cn1860** [comment]
不过，签到没了还是挺遗憾的，哈哈哈。

**Comment by lamquan1220** [comment]
感谢站长，服务真的很稳。高并发也非常稳定！会长期支持。

**Comment by guugg** [comment]
用GitHub登录的，现在不能GitHub登录了怎么办，重置密码出现403

**Reply by BennyThink** [reply]
发邮件给我处理

**Comment by wsqgr** [comment]
竟然和我一个论坛，太丢脸了

**Reply by myvica** [reply]
林子大了什么鸟都有

**Comment by qzydustin** [comment]
c是盗刷信用卡的意思，可以直接说出来

**Comment by ENIAC1729** [comment]
我30刀一般使用够用吗？之前攒的一直都没有舍得用

**Comment by lixiaodong007** [comment]
![image](https://github.com/user-attachments/assets/e5460e62-1883-4f48-9481-056edd65bf7e)
赞👍🏻

**Comment by tianzhuwei** [comment]
白嫖用户路过 感谢提供的稳定服务  稍后会转为付费 再次感谢

**Comment by awayme** [comment]
支持，你根本不用解释这么多，为这些人浪费时间根本不值得。我也是最早的用户，但最早送的额度都没用完，因为后来直接去用官方网页版的claude了。还是要感谢站长做了那么多，哪天我需要付费api的时候，站长你的服务一定是我第一优选。再次感谢
